images/scorecard-test-kuttl/main.go

```diff
@@ -73,17 +63,18 @@ func main() {
 	fmt.Println(string(jsonOutput))
 }
 
-func getTestStatus(tc []*Testcase) (s v1alpha3.TestStatus) {
+func getTestStatus(ts []*Testsuite) (s v1alpha3.TestStatus) {
 
 	// report the kuttl logs when kuttl tests can not be run
 	// (e.g. RBAC is not sufficient)
-	if len(tc) == 0 {
+	if len(ts) == 0 || len(ts[0].Testcase) == 0 {
 		r := v1alpha3.TestResult{}
 		r.Log = getKuttlLogs()
 		s.Results = append(s.Results, r)
 		return s
 	}
 
+	tc := ts[0].Testcase
```

Comment:
I'm not sure there is really a reason to change this logic. My understanding here is that as long as there is a test suite defined, then this function will be called. If the tests can not be run for any reason it will return the kuttl logs.

Reply:
After re-evaluating, I think the above comment still stands. I don't see any real reason to change this logic as I think it may be better for the changes made to be outside of this function. If kuttl fails to run I don't think it should ever make it to this function.